docs/runtime-contract.md

```diff
@@ -125,47 +112,36 @@ the OCI specification as long as:
 - Platforms MAY provide mechanisms for post-mortem viewing of filesystem
   contents from a particular execution. Because containers (particularly failing
   containers) can experience frequent starts, operators or platform providers
-  SHOULD limit the total space consumed by these failures.
-- A container should write its own termination message to `/dev/termination-log`
```

Comment:
I removed it because:

1. It is written from the standpoint of the container. Should the container write the termination message on all Knative platforms (i.e. All installs MUST implement and offer `terminationMessagePolicy`), or just some platforms?
2. We do not have test coverage for this and I couldn't quite find where we actually consume this information and present it back to users in our API. It is not clear to me that we do this at all.
3. Most importantly, this seems like an overly limiting runtime requirement that is not in the critical path at the MUST/SHOULD level and does not add much developer value at the MAY level.

I am not stuck on this one and could be convinced to just rewording it.

Reply:
Yeah -- good catch. I don't think we need to keep this, because it could be a scalability / visibility concern to report these up to the control plane.

In particular, what happens if you have a simultaneous crash of 10k Pods -- aggregation termination messages back to the Revision could be fairly expensive, with each Pod crash causing another attempted update of the Revision. It looks like these don't get aggregated from Pod -> ReplicaSet in k8s, so let's follow that precedent.